frontend_lib/src/appContentFactory.js

```diff
@@ -682,6 +683,51 @@ export function appContentFactory (WrappedComponent) {
       return response
     }
 
+    appContentChangeType = async (content, setState) => {
```

Comment:
this function seems to have a lot of duplicated code

Reply:
if the difference is only content.id or content.content_id, factorize it and add a comment to explain in which case it is .id and .content_id

I'm assuming it is whether it's from app thread or recent activites

Reply:
The comment is missing

Reply:
done